index.js

```diff
@@ -1,6 +1,5 @@
 export * from './src/errors'
 export * from './src/Fold'
-export * from './src/functions'
```

Comment:
I think if we do this we do not expose anywere else `get` and `over` 🤷‍♂️

Reply:
No, because the optics versions are exposed from `operations`, not from `functions`. In fact, I think we shouldn't have exposed `functions` at all, since it has only operations which do not belong to the domain of "optics", like `compose` (which we don't use anywhere) and `curry`.